Mathlib/Data/Finset/Prod.lean

```diff
@@ -93,6 +93,15 @@ theorem product_subset_product_left (hs : s ⊆ s') : s ×ˢ t ⊆ s' ×ˢ t :=
 theorem product_subset_product_right (ht : t ⊆ t') : s ×ˢ t ⊆ s ×ˢ t' :=
   product_subset_product (Subset.refl _) ht
 
+theorem prodMap_image_product {δ : Type*} [DecidableEq β] [DecidableEq δ]
+    (f : α → β) (g : γ → δ) (s : Finset α) (t : Finset γ) :
+    (s ×ˢ t).image (Prod.map f g) = s.image f ×ˢ t.image g :=
+  mod_cast Set.prodMap_image_prod f g s t
+
+theorem prodMap_map_product {δ : Type*} (f : α ↪ β) (g : γ ↪ δ) (s : Finset α) (t : Finset γ) :
```

Comment:
Annoying that apparently `prodMap` can refer both to `Prod.map` and `prodMap`, but I think it's correct according to the current convention

Reply:
I think that a collision between an unbundled and a bundled form of the same operation is not that bad.